### PR TITLE
updated docs config

### LINKED ISSUE #894: Change docs config parameters
Change the docs-v1 mentions of "Common" to become "k8ssandra-operator (current)" and "v1" to become "K8ssandra v1 (deprecated)". Occurrences of "v2" have to be removed. In k8ssandra-operator repo.

### PR DESCRIPTION
**Pull request by RachanKaur**
**What this PR does**:

**Which issue(s) this PR fixes**:
Fixes #894 

**Checklist**
- [ ] Changes manually tested
- [ ] Automated Tests added/updated
- [ ] Documentation added/updated
- [ ] CHANGELOG.md updated (not required for documentation PRs)
- [ ] CLA Signed:  [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

**Review by adejanovski — CHANGES_REQUESTED** [review]
As I'm testing this, I see that we need some consistency between the npm targets and the profiles we have available.
We have development and staging-v2 as environment configs:
Capture d’écran 2023-02-22 à 10 22 14

but these as targets:
```
   "build:staging": "hugo --environment staging",
    "build:production": "hugo --environment production",
    "deploy:staging": "hugo deploy --environment staging --target staging --verbose",
    "deploy:production": "hugo deploy --environment production --target production --verbose"
```

the`config/staging-v2` folder needs to be renamed to staging to match the npm targets and `config/production-v2` must be reintroduced and renamed as `config/production` with the same changes that you're making to the headers (replace Common and v1)

**Comment by RachanKaur** [comment]
> As I'm testing this, I see that we need some consistency between the npm targets and the profiles we have available. We have development and staging-v2 as environment configs: Capture d’écran 2023-02-22 à 10 22 14
> 
> but these as targets:
> 
> ```
>    "build:staging": "hugo --environment staging",
>     "build:production": "hugo --environment production",
>     "deploy:staging": "hugo deploy --environment staging --target staging --verbose",
>     "deploy:production": "hugo deploy --environment production --target production --verbose"
> ```
> 
> the`config/staging-v2` folder needs to be renamed to staging to match the npm targets and `config/production-v2` must be reintroduced and renamed as `config/production` with the same changes that you're making to the headers (replace Common and v1)

@adejanovski production-v2 refers to url: 
```baseURL = "https://docs-v2.k8ssandra.io/"
url_latest_version = "https://docs-v2.k8ssandra.io"```
Shouldn't that be referring to 
```
baseURL = "https://docs.k8ssandra.io/"
url_latest_version = "https://docs.k8ssandra.io"
```
?

**Comment by adejanovski** [comment]
> Shouldn't that be referring to
> 
> ```
> baseURL = "https://docs.k8ssandra.io/"
> url_latest_version = "https://docs.k8ssandra.io"
> ```
> 
> ?

Yes, it should now 👍

File: docs/config/production/config.toml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+baseURL = "https://docs.k8ssandra.io/"
+url_latest_version = "https://docs.k8ssandra.io"
+
+[[params.versions]]
+  version = "k8ssandra-operator (current)"
+  fullversion = "k8ssandra-operator"
+  githubbranch = "docs"
+  url = "https://docs-temp.k8ssandra.io"
```
Comment (adejanovski) [inline docs/config/production/config.toml:8]:
```suggestion
  url = "https://docs.k8ssandra.io"
```

Where did you get `docs-temp` from?

**Review by adejanovski — CHANGES_REQUESTED** [review]

File: docs/config/production/config.toml (hunk 1 above)
Comment (RachanKaur) [reply docs/config/production/config.toml:8]: it was in production-v2. this is what i was going to ask it doesn't seem relevant anymore?

**Review by adejanovski — APPROVED** [review]

**Pull request merged**